site.yml

```diff
@@ -78,6 +78,22 @@
     - role: percona-backup
       tags: ['infra', 'percona']
 
+- name: install mongodb server
+  gather_facts: force
+  hosts: mongo_db
+  roles:
+    - role: mongodb-server
+      tags: ['mongodb', 'mongodb-server']
+      when: ceilometer.enabled|bool
+
+- name: install mongodb arbiter
+  gather_facts: force
```

Comment:
the compute node isn't a server, just an arbiter. so we needed to split up into two plays for the two different host groups - similar to percona